extraction/pom.xml

```diff
@@ -2,147 +2,235 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <parent>
```

Comment:
you broke the 2 space indents here, please fix that (check that your Intellij is configured correctly for Java and XML)